subsys/net/openthread/rpc/server/ot_rpc_netdata.c

```diff
@@ -18,58 +18,35 @@
 static void ot_rpc_cmd_netdata_get(const struct nrf_rpc_group *group, struct nrf_rpc_cbor_ctx *ctx,
 				   void *handler_data)
 {
-	struct nrf_rpc_cbor_ctx rsp_ctx;
-	otError error = OT_ERROR_NONE;
 	bool stable;
-	bool decoded_ok;
-	bool buff_allocated = false;
-	uint8_t length = 0;
-
-	decoded_ok = zcbor_bool_decode(ctx->zs, &stable);
-
-	if (!decoded_ok) {
-		error = OT_ERROR_INVALID_ARGS;
-		goto exit;
-	}
-
-	decoded_ok = zcbor_uint_decode(ctx->zs, &length, sizeof(length));
-	if (!decoded_ok) {
-		error = OT_ERROR_INVALID_ARGS;
-		goto exit;
-	}
-
-	nrf_rpc_cbor_decoding_done(group, ctx);
+	uint8_t length;
+	otError error;
+	uint8_t netdata[UINT8_MAX];
```

Comment:
Is it good idea to consume stack memory here. Probably heap usage will be more safe. It helps to avoid stack overflow.

Reply:
I thought 255 bytes is not large enough to be worry but I can switch to the heap.

Reply:
Done